src/ProductConstructionService/ProductConstructionService.SubscriptionTriggerer/SubscriptionTriggererConfiguration.cs

```diff
@@ -39,3 +43,20 @@ public static HostApplicationBuilder ConfigureSubscriptionTriggerer(
         return builder;
     }
 }
+
+internal class FakeWorkItemProducer<T> : IWorkItemProducer<T> where T : WorkItem
+{
+    public Task DeleteWorkItemAsync(string messageId, string popReceipt) => throw new NotImplementedException();
+
+    public Task EnqueueAsync(T workItem)
+    {
+        return Task.CompletedTask;
+    }
+
+    public Task<SendReceipt> ProduceWorkItemAsync(T payload, TimeSpan delay = default) => throw new NotImplementedException();
```

Comment:
This should not throw but just be a no-op

Reply:
Ah ofc, my bad, I thought I implemented it above, my VS was acting weird while my laptop was overheating...